bamboo/unit_tests/test_unit_lbann2_reload.py

```diff
@@ -37,33 +42,49 @@ def skeleton_lbann2_reload(cluster, executables, dir_name, compiler_name,
     # Run to checkpoint, printing weights to files.
     output_file_name = '%s/bamboo/unit_tests/output/lbann2_checkpoint_%s_output.txt' % (dir_name, compiler_name)
     error_file_name  = '%s/bamboo/unit_tests/error/lbann2_checkpoint_%s_error.txt' % (dir_name, compiler_name)
-    ckpt_dir = 'ckpt_lbann2_reload/lbann2_ckpt_{c}'.format(c=compiler_name)
+    ckpt_dir = os.path.join(ckpt_base_dir,'lbann2_ckpt_{c}'.format(c=compiler_name))
     command = tools.get_command(
-        cluster=cluster, executable=lbann2, num_nodes=1, num_processes=2,
-        dir_name=dir_name,
+        cluster=cluster,
+        executable=lbann2,
+        num_nodes=1,
+        num_processes=2,
+        ckpt_dir=ckpt_dir,
         data_filedir_default='/p/lscratchh/brainusr/datasets/MNIST',
-        data_reader_name='mnist', data_reader_percent=data_reader_percent,
-        ckpt_dir=ckpt_dir, model_folder='tests',
-        model_name='lenet_mnist_ckpt', num_epochs=2, optimizer_name='sgd',
+        data_reader_name='mnist',
+        data_reader_percent=0.005,
```

Comment:
Since this is repeated multiple times, I would store it in a variable, so if a change is necessary then we only have to do it in one place.

Reply:
Fixed.